modules/ROOT/pages/create-outbound-message-flow.adoc

```diff
@@ -1,12 +1,12 @@
 = Create and Configure an Outbound Message Flow
 
-Create an outbound message flow to receive application messages from your backend systems, transform and send the translated EDI message to your trading partners.
+Create an outbound message flow to receive application messages from your backend systems, transform, and then send the translated EDI message to your trading partners.
 
-After you create the outbound message flow, you configure the receive endpoint, source message type, target message type, and target endpoint. 
+After you create the outbound message flow, you configure the receive endpoint, source message type, target message type, and target endpoint.
 
 == Create an Outbound Message Flow
 
-. From your Anypoint Partner Manager *Sandbox* or *Production* environment, select *Message Flows* in the left navigation menu. 
+. From your Anypoint Partner Manager *Sandbox* or *Production* environment, select *Message Flows* in the left navigation menu.
```

Comment:
We are referring to the environment in general, not the UI section label, so...:
```suggestion
. From your Anypoint Partner Manager sandbox or production environment, select *Message Flows* in the left navigation menu.
```
or name the UI labels and do not call them "environments."